c++/src/kj/common.h

```diff
@@ -1314,7 +1314,7 @@ inline T* readMaybe(T* ptr) { return ptr; }
 
 #define KJ_IF_MAYBE(name, exp) if (auto name = ::kj::_::readMaybe(exp))
 
-#if __GNUC__
+#if __GNUC__ || defined(__clang__)
```

Comment:
In KJ preprocessor code we usually rely on the fact that undefined symbols are treated as falsy.

```suggestion
#if __GNUC__ || __clang__
```